client/src/bin/space-cli.rs

```diff
@@ -54,6 +54,15 @@ pub struct Args {
     /// Spaced RPC URL [default: based on specified chain]
     #[arg(long)]
     spaced_rpc_url: Option<String>,
```

Comment:
lets rename this to `rpc_url` without `spaced_` prefix so its consistent with `rpc_` cookie, user, and pass.